db/migrate/20161107113656_clean_up_community_currencies.rb

```diff
@@ -0,0 +1,11 @@
+class CleanUpCommunityCurrencies < ActiveRecord::Migration
+  def up
+    sql = 'UPDATE communities c
+           SET c.currency = "EUR"
+           WHERE c.currency IS NULL'
+    exec_update(sql, "Change NULL currencies to EUR", [])
+
+    change_column_default :communities, :currency, "EUR"
+    change_column_null :communities, :currency, false
```

Comment:
These could be splitted to another migration, because these can be rolled back easily. In addition to that, maybe we shouldn't add the default value? Isn't that a programming error if we add a new community without currency? Maybe in that case it's just better to crash.